forem/forem
# Migrate ImageResizer to Images::Optimizer

## What type of PR is this? (check all applicable)

## Description
A continuation of https://github.com/forem/forem/pull/9652 and following feedbacks.

## Related Tickets & Documents
https://github.com/forem/forem/pull/9652

## QA Instructions, Screenshots, Recordings
1. Startup the server locally
2. Browse around to make ensure all images are loaded properly.

## Added tests?
- [x] yes, updated the specs

## Added to documentation?

## [optional] Are there any post deployment tasks we need to perform?
Keep close attention to production
